app/src/main/java/com/pvp/app/ui/screen/decoration/Decoration.kt

```diff
@@ -44,13 +45,18 @@ private fun DecorationCard(
             .padding(8.dp)
             .clip(MaterialTheme.shapes.medium)
             .background(
-                color = if (holder.applied) {
-                    MaterialTheme.colorScheme.surfaceContainer.darken(0.15f)
-                } else {
-                    MaterialTheme.colorScheme.surfaceContainer
-                },
+                color = MaterialTheme.colorScheme.surfaceContainer.darken(0.2f),
                 shape = MaterialTheme.shapes.medium
             )
+            .let {
```

Comment:
`Modifier.then` exists for this purpose, which appends specified Modifier onto current Modifier instance. But this works too, may be less code and similar readability.